internal/generated/snippets/speech/apiv1/AdaptationClient/create_custom_class.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by cloud.google.com/go/internal/gapicgen/gensnippets. DO NOT EDIT.
+
+package speech_v1
+
+// [START speech_v1_generated_Adaptation_CreateCustomClass_sync]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	speech "cloud.google.com/go/speech/apiv1"
+	"google.golang.org/api/option"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1"
+)
+
+func createCustomClass(w io.Writer, parent, customClassId string) error {
+	// The custom class parent element
+	// parent = "projects/[PROJECT]/locations/us"
+	// The id for the custom class
+	// customClassId = "passengerships"
+
+	ctx := context.Background()
+	c, err := speech.NewAdaptationClient(ctx,
+		option.WithEndpoint("us-speech.googleapis.com:443"))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+	customClass := &speechpb.CustomClass{
+		Items: []*speechpb.CustomClass_ClassItem{
+			{
+				Value: "Titanic",
+			},
+			{
+				Value: "RMS Queen Mary",
+			},
+		},
+	}
+
+	req := &speechpb.CreateCustomClassRequest{
+		Parent:        parent,
+		CustomClassId: customClassId,
+		CustomClass:   customClass,
+	}
+	fmt.Fprintln(w, "Calling the CreateCustomClass operation.")
+	resp, err := c.CreateCustomClass(ctx, req)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintln(w, "A Custom Class with the following name has been created.")
+	fmt.Fprintln(w, resp.Name)
+	fmt.Fprintln(w, "The Custom class contains the following items.")
+	for _, item := range resp.Items {
+		fmt.Fprintln(w, item.Value)
+	}
```

Comment:
As per the config, I believe this code should look like (pardon my Go abilities)
```suggestion
        itemsList := createdCustomClass.Items
	for _, item := range itemsList {
		fmt.Fprintln(w, item.Value)
	}
```

Reply:
Fixed, thanks!